dataactbroker/README.md

```diff
@@ -1,2746 +0,0 @@
-# The DATA Act Broker Application Programming Interface (API)
```

Comment:
Could we add/keep a super basic README for the dataactbroker directory to point them to the docs directory? Something like

```
Documentation on the Broker API can be found in the [docs directory](./doc/))
```